runtime/dotnet/azurefunctions/Microsoft.BotFramework.Composer.Functions.csproj

```
@@ -13,18 +13,18 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Functions.Extensions" Version="1.0.0" />
-    <PackageReference Include="Microsoft.Bot.Builder" Version="4.9.0" />
-    <PackageReference Include="Microsoft.Bot.Builder.AI.Luis" Version="4.9.0" />
-    <PackageReference Include="Microsoft.Bot.Builder.AI.QnA" Version="4.9.0" />
-    <PackageReference Include="Microsoft.Bot.Builder.ApplicationInsights" Version="4.9.0" />
-    <PackageReference Include="Microsoft.Bot.Builder.Azure" Version="4.9.0" />
-    <PackageReference Include="Microsoft.Bot.Builder.Dialogs.Adaptive" Version="4.9.0" />
-    <PackageReference Include="Microsoft.Bot.Builder.Dialogs.Debugging" Version="4.9.0-preview" />
-    <PackageReference Include="Microsoft.Bot.Builder.Dialogs.Declarative" Version="4.9.0" />
-    <PackageReference Include="Microsoft.Bot.Builder.Integration.ApplicationInsights.Core" Version="4.9.0" />
-    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.9.0" />
-    <PackageReference Include="Microsoft.Bot.Builder.Dialogs" Version="4.9.0" />
-    <PackageReference Include="Microsoft.Bot.Connector" Version="4.9.0" />
+    <PackageReference Include="Microsoft.Bot.Builder" Version="4.9.1" />
+    <PackageReference Include="Microsoft.Bot.Builder.AI.Luis" Version="4.9.1" />
+    <PackageReference Include="Microsoft.Bot.Builder.AI.QnA" Version="4.9.1" />
+    <PackageReference Include="Microsoft.Bot.Builder.ApplicationInsights" Version="4.9.1" />
+    <PackageReference Include="Microsoft.Bot.Builder.Azure" Version="4.9.1" />
+    <PackageReference Include="Microsoft.Bot.Builder.Dialogs.Adaptive" Version="4.9.1" />
+    <PackageReference Include="Microsoft.Bot.Builder.Dialogs.Debugging" Version="4.9.1-preview" />
+    <PackageReference Include="Microsoft.Bot.Builder.Dialogs.Declarative" Version="4.91" />
```

This doesn't sound right.

---

The web app project got it right, suggestion below:

```suggestion
    <PackageReference Include="Microsoft.Bot.Builder.Dialogs.Declarative" Version="4.9.1" />
```